paddle/fluid/operators/select_op_helper.h

```diff
@@ -26,8 +26,12 @@ namespace operators {
 // Returns the integer in mask whose numel must be 1. The integer means the
 // selected branch number.
 inline int GetBranchNumber(const framework::LoDTensor &mask) {
-  PADDLE_ENFORCE_EQ(mask.numel(), 1,
-                    "Mask in SelectOutputOp must have numel 1.");
+  PADDLE_ENFORCE_EQ(
+      mask.numel(), 1,
+      platform::errors::InvalidArgument(
+          "Mask in SelectInputOp or SelectOutputOp must have numel 1. "
+          "But received %d, and it's shape is [%s].",
```

Comment:
`The numel of Mask in SelectInputOp or SelectOutputOp must be 1.`

Reply:
Done